native/server/src/main.rs

```diff
@@ -22,7 +25,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     println!("Brongnal Server listening at: {server_addr}");
 
-    let controller = BrongnalController::new(Box::new(MemoryStorage::default()));
+    let mut db_path = PathBuf::from(std::env::var("DB").unwrap_or(String::from("db")));
```

Comment:
Why have this as mutable, instead of creating with "brongnal.db3" as part of the original PathBuf?

Reply:
Because this is better

Reply:
Fixed